<?xml version="1.0"?>
<launch>
    <arg name="qr_x" default="0.0" />
    <arg name="qr_y" default="0.0" />
    <arg name="qr_z" default="0.0" />
    <arg name="qr_qx" default="0.0" />
    <arg name="qr_qy" default="0.0" />
    <arg name="qr_qz" default="0.0" />
    <arg name="qr_qw" default="1.0" />

    <node pkg="approach_locate" type="approach_locate_init_node" name="locate" required="true"
        output="screen">
        <param name="qr_x" value="$(arg qr_x)" />
        <param name="qr_y" value="$(arg qr_y)" />
        <param name="qr_z" value="$(arg qr_z)" />
        <param name="qr_qx" value="$(arg qr_qx)" />
        <param name="qr_qy" value="$(arg qr_qy)" />
        <param name="qr_qz" value="$(arg qr_qz)" />
        <param name="qr_qw" value="$(arg qr_qw)" />
    </node>
</launch>